mobile/app/lib/cothority/network/proto.ts

```diff
@@ -172,21 +173,22 @@ export class ServerIdentity extends Message<ServerIdentity> {
      */
     static urlToWebsocket(url: string): string {
         const urlParser = new URL(url);
+        let proto = "";
         switch (urlParser.protocol) {
             case "http:": {
-                urlParser.protocol = "ws:";
+                proto = "ws:";
                 break;
             }
             case "https:": {
-                urlParser.protocol = "wss:";
+                proto = "wss:";
                 break;
             }
             default : {
                 throw new Error("The url field should use either 'http:' or 'https:', but we found "
                     + urlParser.protocol);
```

Comment:
```suggestion
                     + proto);
```

Reply:
why? This is the error-message, so `proto` is not set.

Reply:
Oups, you're correct